lib/verify.go

```diff
@@ -129,6 +155,20 @@ func VerifyChain(certs []*x509.Certificate, ocspStaple []byte, dnsName, caPath s
 	chains, err := certs[0].Verify(opts)
 	if err != nil {
 		result.Error = fmt.Sprintf("%s", err)
+
+		if len(intermediates.Subjects()) == 0 {
+			// No intermediates found, maybe broken chain. Let's try AIA fetching?
+			intermediate := fetchAIA(certs[0])
```

Comment:
We're only trying the first one in the chain? I guess it's pretty rare to have more than 3 certs in a chain.